### DISCUSSION TITLE
Where to find gsp_log_*.bin to print GSP-RM log on dmesg?

### DISCUSSION THREAD

**Discussion by karim-manaouil** [NVIDIA/open-gpu-kernel-modules#490] [Q&A]
Hi!

I have NVreg_EnableGpuFirmwareLogs enabled to see the log from GSP-RM, but I am getting an error when trying to load gsp_log_tu10x.

`NVRM RmInitAdapter: Failed to load gsp_log_*.bin, no GSP-RM logs will be printed (non-fatal)`

I checked in `/lib/firmware/nvidia` and I can't find such a firmware file. 

Where can I find gsp_log_tu10x firmware blob for 530.30.02?

Cheers!
Karim

**Comment by ttabi** [comment]
The gsp_log files are not distributed.  NVreg_EnableGpuFirmwareLogs is not meant to be enabled, because it requires the log file which isn't available.

Why did you enable this option?

**Reply by karim-manaouil** [reply]
Hi Timur. Thanks for your reply!

I am leveraging the open kernel module to do research on GPUs and the RM log would be very helpful to understand certain features and debug my code. Would it be possible to distribute that in the near future? Or make an exception case for us as we are a research lab (University of Edinburgh, UK)?

Cheers
karim

**Comment by ttabi** [comment]
Sorry, there are no plans to make the log file available to customers.

**Reply by karim-manaouil** [reply]
Sad, but thanks for the answers Timur!
